test/1_stdlib/Runtime.swift

```diff
@@ -2116,7 +2031,8 @@ Reflection.test("TupleMirror/NoLeak") {
       var output = ""
       dump(tuple, &output)
     }
-    expectEqual(0, nsObjectCanaryCount)
+    // Disabled pending SR-426
+    // expectEqual(0, nsObjectCanaryCount)
```

Comment:
Thanks to Jesse Rusak, fixes for both this bug and the other referenced bug have been pushed. This unit test needs to be turned back on before the PR can progress, though.